content/docs/forms.md

```diff
@@ -9,23 +9,24 @@ redirect_from:
   - "docs/forms-zh-CN.html"
 ---
 
-HTML form elements work a little bit differently from other DOM elements in React, because form elements naturally keep some internal state. For example, this form in plain HTML accepts a single name:
+रियेक्ट मै HTML फॉर्म के तत्व DOM के तत्वो से थोड़ा भिन्न तरीके से काम करते है, क्योंकि फार्म के तत्व स्वाभाविक रूप से कुछ आंतरिक स्थिति रखते हैं, जैसे की, सादे HTML में यह फॉर्म एक ही नाम स्वीकार करेगा:
 
 ```html
 <form>
   <label>
-    Name:
-    <input type="text" name="name" />
+    Name
+    <input  type ="text" name ="name" />
```

Comment:
`<input  type ="text" name ="name" />`
remove unwanted extra spaces, keep it same as in english